scripts/helm/publish-chart-yaml.sh

```diff
@@ -108,6 +112,7 @@ Options:
   --override-index       <latest_version>         Override the latest chart version from the published chart's index.
   --index-file           <index_yaml>             Use the provided index.yaml instead of fetching from the git branch.
   --override-chart       <version> <app_version>  Override the Chart.yaml version and app version.
+  --date-time            <date-time>              The date-time in the format +"$DATE_TIME_FMT".
```

Comment:
What's the significance of the '+' before "$DATE_TIME_FMT" ?

Reply:
Oh that's just how the date tool works AIUI